cumulusci/tasks/bulkdata/generate.py

```diff
@@ -209,7 +227,12 @@ def _split_dependencies(self, objs, dependencies):
                                 other_obj, ", ".join(dependencies[obj][other_obj])
                             )
                         )
-                raise BulkDataException("Cannot complete mapping")
+                choice = click.prompt(
+                    "Which object should we load first?",
+                    type=click.Choice(list(objs_remaining)),
+                    show_choices=True,
+                )
+                objs_without_deps = [choice]
```

Comment:
Generally, tasks should not assume they're running in the CLI, so should avoid using imports from `click`. That said, we don't (yet?) have a generic framework for tasks to prompt for input in a runtime-environment-agnostic way, and we don't particularly need for this task to be run other than in the CLI, so I'm inclined to let it stand for now.

Reply:
Do we have any established convention for saying "Task X requires interaction"? It's a really important point, and it might be good to call out this expectation.

Reply:
No...it's probably a good idea, because then we could at least make the metadeploy publish task throw an error if you try to publish a flow that includes one of them. But I think the ideal end state is one where inputs like this are included in the list of task options so there's a chance of specifying them completely enough in cumulusci.yml that interaction isn't required.